core/configuredivecomputerthreads.cpp

```diff
@@ -1169,10 +1203,14 @@ static dc_status_t read_ostc3_settings(dc_device_t *device, DeviceDetails &devic
 	deviceDetails.customText = ar.trimmed();
 	EMIT_PROGRESS();
 
+	if (rc == DC_STATUS_SUCCESS)
```

Comment:
This check is redundant. If rc is anything but DC_STATUS_SUCCESS , we would already have returned.,